apps/test/unit/SoundsTest.js

```diff
@@ -1,11 +1,11 @@
 import {expect} from '../util/deprecatedChai';
 import Sounds from '@cdo/apps/Sounds';
 import sinon from 'sinon';
+import winMp3 from '!!file-loader!../audio/assets/win.mp3';
 
 describe('Sounds', () => {
   const sounds = new Sounds();
-  const sourceURL =
-    'studio.code.org/api/v1/sound-library/KyZOBksdJiQSlvoiOzFGpryJiMexdfks/category_ui/click1.mp3';
```

Comment:
This URL 404s, and where PhantomJS left its sound in a "still loading" state so that the test would pass, ChromeHeadless resolves the 404 right away and leaves the sound in a "load failed" state.